erpnext/accounts/doctype/payment_request/payment_request.py

```diff
@@ -235,6 +254,19 @@ def on_payment_authorized(self, status=None):
 
 			return redirect_to
 
+	def create_subscription(self, payment_provider, gateway_controller, data):
```

Comment:
Caller??

Reply:
It is located in Frappe (in order to keep Stripe in Frappe):
https://github.com/frappe/frappe/blob/develop/frappe/templates/pages/integrations/stripe_checkout.py